lib/tasks/traject.rake

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'psulib_blacklight/solr_manager'
+require 'psulib_blacklight/data_manager'
+
+require 'net/http'
+
+namespace :traject do
+  # We only run this task in kubernetes. Hide it if the WEBHOOK_URL is not set
+  if ENV['TRAJECT_WEBHOOK_URL']
+    desc 'Creates a new collection, then fires off traject job to index into that collection'
+    task :iterate_and_import, [:path] => [:environment] do |_task, args|
+      path = args['path'] || 'full_extracts'
+      traject_webhook_url = ENV['TRAJECT_WEBHOOK_URL'] || raise('TRAJECT_WEBHOOK_URL is missing')
```

Comment:
I don't think it's possible for `TRAJECT_WEBHOOK_URL` to be missing due to the check at line 10, right?

Reply:
correct. line 10 was added later, when I realized that it's not terribly useful to have these tasks outside of k8s